includes/class-newspack-newsletters-renderer.php

```diff
@@ -224,6 +224,30 @@ private static function get_colors( $block_attrs ) {
 		return $colors;
 	}
 
+	/**
+	 * Get spacing value.
+	 *
+	 * @param string $value Spacing value.
+	 *
+	 * @return string Spacing value.
+	 */
+	private static function get_spacing_value( $value ) {
+		$presets = [
+			'50' => 'clamp( 1.25rem, 1rem + 0.8333vw, 1.5rem )',
+			'60' => 'clamp( 1.5rem, 0.75rem + 2.5vw, 2.25rem )',
+			'70' => 'clamp( 1.75rem, 0.12rem + 5.4333vw, 3.38rem )',
+			'80' => 'clamp( 2rem, -1.06rem + 10.2vw, 5.06rem )',
+		];
+		if ( 0 === strpos( $value, 'var' ) ) {
+			$preset = end( explode( '|', $value ) );
```

Comment:
Nit: the use of the `explode` function inside `end` is causing a PHP notice.

```suggestion
			$preset_key = explode( '|', $value );
			$preset     = end( $preset_key );
```